### DISCUSSION TITLE
Can you update the conda package?

### DISCUSSION THREAD

**Discussion by cacharle** [py-pdf/fpdf2#311] [Q&A] [answered]
I don't know who is maintaining the [anaconda package][1] or even if it's a good idea to update it but is there any plan of updating it to the latest version of fpdf2?

I don't have access to pip and I would like to use the document outline feature (which was released in 2.3.3).

[1]: https://anaconda.org/conda-forge/fpdf2

**Comment by Lucas-C** [comment, marked as answer]
As `fpdf2` maintainer, neither do I know who is maintaining the conda package 😅

After searching a little bit, I see there are already PRs open there:
https://github.com/conda-forge/fpdf2-feedstock#updating-fpdf2-feedstock

The maintainer seems to be a bot: @conda-forge/rpaframework-core

I haven't found in the docs who is supposed to approved those PRs...
https://conda-forge.org/docs/maintainer/updating_pkgs.html

**Reply by cacharle** [reply]
Thank you for the information, I left a comment on the PR I'm interested in, maybe this will notify someone and it'll get merged.
